config.lua

```diff
@@ -25,24 +29,66 @@ config({'development', 'production'}, {
     -- read & write permissions to that path.
     store_path = 'store',
 
-     -- for sending email
-    mail_server = "",
-    mail_user     = "",
-    mail_password = "",
-    mail_from = "",
-    mail_footer = ""
+    -- for sending email
+    mailgun = {
+        domain = os.getenv('MAILGUN_DOMAIN'),
+        api_key = os.getenv('MAILGUN_API_KEY')
+    },
+    mail_user = os.getenv('MAIL_SMTP_USER'),
+    mail_password = os.getenv('MAIL_SMTP_PASSWORD'),
+    mail_server = os.getenv('MAIL_SMTP_SERVER'),
+    mail_from_name = 'Snap!Cloud',
+    mail_from = "noreply@snap-cloud.cs10.org",
+    mail_footer = "This is a test",
+
+    measure_performance = true,
+
+    cors_domains = {
+        'localhost:' .. port,
+        'snap.berkeley.edu'
+    }
 })
 
 config('production', {
-    site_name = 'Snap Cloud',
+    use_daemon = 'on',
```

Comment:
What does this do?

Reply:
That sets the nginx dameon setting. Typically it's off for development, so you can see the logs more easily, but needs to be on for production.  It's exposed as `${{USE_DAEMON}}` at the top of nginx.conf